doc/source/guidelines/PyMAPDL_WSL_Guide.rst

```diff
@@ -0,0 +1,423 @@
+If you are having problems after setting the firewall rules, you might have to disable Windows Firewall for the WSL ethernet virtual interface.
+This might pose some unknown side effects and security risk so use it with caution.
+See `Disabling Firewall on WSL Ethernet`_
+
+
+Setting up an environmental variable in WSL that points to Windows Host License Server
+---------------------------------------------------------------------------------------
+
+Windows host IP is given in the WSL file ``/etc/hosts`` before the name ``host.docker.internal``.
+
+
+.. note:: This ``host.docker.internal`` definition might not be available if docker is not installed. 
+
+
+**Example /etc/hosts/ file**
+
+.. code-block:: bash
+   :emphasize-lines: 7
+
+   # This file was automatically generated by WSL. To stop automatic generation of this file, add the following entry to /etc/wsl.conf:
+   # [network]
+   # generateHosts = false
+   127.0.0.1       localhost
+   127.0.1.1       AAPDDqVK5WqNLve.win.ansys.com   AAPDDqVK5WqNLve
+
+   192.168.0.12    host.docker.internal
+   192.168.0.12    gateway.docker.internal
+   127.0.0.1       kubernetes.docker.internal
+
+   # The following lines are desirable for IPv6 capable hosts
+   ::1     ip6-localhost ip6-loopback
+   fe00::0 ip6-localnet
+   ff00::0 ip6-mcastprefix
+   ff02::1 ip6-allnodes
+   ff02::2 ip6-allrouters
+
+You can add the next lines to you WSL ``~/.bashrc`` file to create an environment variable with that IP:
+
+.. code:: bash
+
+    winhostIP=$(grep -m 1 host.docker.internal /etc/hosts | awk '{print $1}')
+    export ANSYSLMD_LICENSE_FILE=1055@$winhostIP
+
+
+Running ANSYS on a local docker
+********************************
+
+To run a docker image, you need to follow all the previous steps detailed in `Running ANSYS on WSL`_ .
+
+Additionally, to run a docker PyMAPDL image, use the next command:
+
+.. code:: pwsh
+
+    docker run -e ANSYSLMD_LICENSE_FILE=1055@host.docker.internal --restart always --name mapdl -p 50053:50052 docker.pkg.github.com/pyansys/pymapdl/mapdl -smp > log.txt
+
+Successive runs should restart the container or just delete it and rerun it using:
+
+.. code:: pwsh
+
+    docker stop mapdl
+    docker container prune
+
+    docker run -e ANSYSLMD_LICENSE_FILE=1055@host.docker.internal --restart always --name mapdl -p 50053:50052 docker.pkg.github.com/pyansys/pymapdl/mapdl -smp > log.txt
+
+
+This will create a log file (``log.txt``) in your current directory location.
+
+
+.. note:: Please make sure your ports (``50053``) are open in your firewall.
+
+It is recommended to do a script file (batch ``'.bat'`` or powershell ``'.ps'`` files) to run the above commands, all at once.
+
+Please do notice that we are mapping the iWSL nternal gRPC port (``50052``) to a different Windows host port (``50053``) to avoid ports conflicts.
```

Comment:
```suggestion
Notice that we are mapping the WSL internal gRPC port (``50052``) to a different Windows host port (``50053``) to avoid ports conflicts.
```